unstructured/ingest/cli/cmds/astra.py

```diff
@@ -0,0 +1,61 @@
+import typing as t
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.cli.interfaces import CliConfig, Dict
+from unstructured.ingest.connector.astra import AstraDBWriteConfig, SimpleAstraDBConfig
+
+
+@dataclass
+class AstraDBCliConfig(SimpleAstraDBConfig, CliConfig):
+    @staticmethod
+    def get_cli_options() -> t.List[click.Option]:
+        options = [
+            click.Option(
+                ["--token"],
+                required=True,
+                type=str,
+                help="Astra DB Token with access to the database.",
+            ),
+            click.Option(
+                ["--api_endpoint"],
+                required=True,
+                type=Dict(),
```

Comment:
Wasn't sure about this but I input the endpoint as a string in my environment. So this should be `str` I believe.

Reply:
Yep, this was likely something copied from another parameter and i missed updating it. It's definitely a str - the Astra DB portal gives you the string (on a side note, that string does encode a fair amount of information such that it probably could be a str, but yeah, our client libraries assume its the fully qualified web url so this should absolutely be a str). I've updated this as well.